### DISCUSSION TITLE
Signing up for Student Dev Pack requires full bio but I already have 1

### DISCUSSION THREAD

**Discussion by tobycm** [community/community#41966] [GitHub Education]
Like title, I already have a full bio but GitHub keeps asking me to fill it in.
![image](https://user-images.githubusercontent.com/62174797/208233762-e12823d4-bd48-4a6a-b22e-b07daaf0082b.png)

Student ID censored of course

**Comment by tobycm** [comment]
I did try contacting support but they ghosted me for 3 weeks.

**Comment by jfullstackdev** [comment]
just commenting so that you also know that others are seeing this post

there are Staff members here that can accelerate the request but there is no assurance whether they will be responding here as they are dealing with too many requests

**Comment by samus-aran** [comment]
Hi @tobycm 👋 

We're happy you're here! Let's see how we can assist. Often this fairly common pre-processing application hint is because it detects differences in your evidence to your profile. Here are a few tips to adjust your profile and application to have an increased chance of a successful application.

1. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic email being at the top. If you don't have an Academic email or Student ID do try and acquire evidence that is positioned higher rather than lower in that option list.
2. As you can imagine GitHub processes a number of applications - we do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification looks at your GitHub username, email and profile name (and other aspects). Do these match in your evidence? 

Hope these help. 👍  Let us know how you get on.

**Reply by tobycm** [reply]
Hi! Thank you so much for such a helpful response.

Before this post, I actually selected Student ID for the proof type but I have an error and already made a community post about that. Sadly there's aren't any answers for it so I choose Other proof type instead and put "Student ID" in.

About the profile, I don't want my real name to be in the profile and are there any alternatives or workarounds? (Maybe I just need to contact support again...)
